src/platform/sound_device.c

```diff
@@ -17,8 +18,9 @@
 #define MAX_CHANNELS 150
 
 #if SDL_VERSION_ATLEAST(2, 0, 7) 
-#define USE_SDL_AUDIOSTREAM 
+#define USE_SDL_AUDIOSTREAM
 #endif
+#define HAS_AUDIOSTREAM() (platform_version_at_least(2, 0, 7))
```

Comment:
Do we really need this `#define`? We only use it once on line `65`...

Reply:
I created it to keep the 2.0.7 SDL versions together, otherwise there will be one check for 2.0.7 at the top, and one somewhere in the code.

Reply:
Fair enough.